ts/annotations/annotations.src.ts

```diff
@@ -497,6 +505,19 @@ merge(
              */
             visible: true,
 
+            /**
+             * Whether to defer displaying the annotations until the set
+             * duration time has finished. Setting to `'false'` disable
```

Comment:
- Shouldn't be `'false'` -> `false` ? The same below with `true`.
- disable -> disables
- perhaps simply "Setting to `false` renders annotations immediately." ?